content/docs/components-and-props.md

```diff
@@ -15,24 +15,25 @@ redirect_from:
 prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
+Los componentes permiten separar la UI en piezas independientes, reutilizables y pensar en cada pieza de forma aislada.
+Esta página proporciona una introducción a la idea de los componentes.
```

Comment:
This should not be a new line. `Esta página proporciona....` should go right after `de forma aislada.` in line 18.